rest_framework/renderers.py

```diff
@@ -635,6 +635,7 @@ def get_context(self, data, accepted_media_type, renderer_context):
             'view': view,
             'request': request,
             'response': response,
+            'user': request.user,
```

Comment:
Verified. Previously we were relying on the default value of `TEMPLATE_CONTEXT_PROCESSORS`, which includes `django.contrib.auth.context_processors.auth`.

We could add this in to the `TEMPLATES.OPTIONS.context_processors` in `conftest.py`, which would resolve our test cases, but it's better if we pass it explicitly, and not rely on the user settings.